src/Components/Server/src/Circuits/ServerAuthenticationStateProvider.cs

```diff
@@ -15,7 +15,7 @@ public class ServerAuthenticationStateProvider : AuthenticationStateProvider, IH
     /// <inheritdoc />
     public override Task<AuthenticationState> GetAuthenticationStateAsync()
         => _authenticationStateTask
-        ?? throw new InvalidOperationException($"{nameof(GetAuthenticationStateAsync)} was called before {nameof(SetAuthenticationState)}.");
+        ?? throw new InvalidOperationException($"{nameof(GetAuthenticationStateAsync)} was called before {nameof(SetAuthenticationState)}. You cannot call nameof(GetAuthenticationStateAsync)} outside a Blazor context.");
```

Comment:
```suggestion
        ?? throw new InvalidOperationException($"{nameof(GetAuthenticationStateAsync)} was called before {nameof(SetAuthenticationState)}. You cannot call {nameof(GetAuthenticationStateAsync)} outside of a Blazor context.");
```

Reply:
Since we're bikeshedding, "outside of" sounds weird to me, but that's probably because it's a USism and not really used in the UK. However according to https://english.stackexchange.com/a/9701, "outside of" is an informal colloquialism. As such the simpler and more formal "outside" would be a normal tone for an exception message.

Reply:
FWIW, I also found "outside of" jarring.